libguard/include/guard_record.hpp

```diff
@@ -12,6 +12,7 @@ namespace guard
 #define GUARD_MAGIC "GUARDREC" //! "GUARDREC"
 /* From hostboot: src/include/usr/hwas/hwasPlatDeconfigGard.H */
 const uint8_t CURRENT_GARD_VERSION_LAYOUT = 0x1;
+#define INVALID_RECORD_ID 0xFFFFFFFF
```

Comment:
GUARD_RESOLVED?

Reply:
Can discuss on name.

Reply:
Bring it in some forum if you have concern

Reply:
Changing to GUARD_RESOLVED.